teku/src/main/java/tech/pegasys/teku/cli/options/ValidatorKeysOptions.java

```diff
@@ -104,12 +104,23 @@ public class ValidatorKeysOptions {
   private int validatorExternalSignerConcurrentRequestLimit =
       ValidatorConfig.DEFAULT_VALIDATOR_EXTERNAL_SIGNER_CONCURRENT_REQUEST_LIMIT;
 
+  @CommandLine.Option(
+      names = {"--allow-no-loaded-keys-enabled"},
```

Comment:
Allow and Enabhled on the same parameter makes it a bit confusing. Can it be something like: `--allow-no-loaded-keys`?